### ISSUE TITLE
Training the UNet...  /bin/bash: -c: line 0: syntax error near unexpected token `('

### ISSUE THREAD

**Issue by inck86** [TheLastBen/fast-stable-diffusion#1307]
**An error occurred when trying to train a model with Huggingface (any model, tried dreamlike, openjourney).
If you do not select a custom model, then "Training the text encoder" passes without errors, but with "Unet" the same error appears**

Training the UNet...

/bin/bash: -c: line 0: syntax error near unexpected token `('
/bin/bash: -c: line 0: `accelerate launch /content/diffusers/examples/dreambooth/train_dreambooth.py                  --stop_text_encoder_training=350      --image_captions_filename      --train_only_unet      --save_starting_step=500      --save_n_steps=0      --Session_dir=/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6      --pretrained_model_name_or_path="/content/stable-diffusion-custom"      --instance_data_dir="/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6/instance_images"      --output_dir="/content/models/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6"      --captions_dir="/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6/captions"      --instance_prompt=""      --seed=806191      --resolution=512      --mixed_precision=fp16      --train_batch_size=1      --gradient_accumulation_steps=1 --gradient_checkpointing      --use_8bit_adam      --learning_rate=5e-06      --lr_scheduler="polynomial"      --lr_warmup_steps=0      --max_train_steps=2100'

Something went wrong
![9c27d8b4-0a1c-4419-bf92-856c27bda7ff](https://user-images.githubusercontent.com/119116007/211171310-07f7141d-cbab-40c0-9b4b-e9ffa8d94e6f.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/119116007/211171632-91b7cd1d-ed84-409f-a839-ff173d0bd227.png)

**Comment by inck86** [comment]
Error without custom model:

Training the UNet...

/bin/bash: -c: line 0: syntax error near unexpected token `('
/bin/bash: -c: line 0: `accelerate launch /content/diffusers/examples/dreambooth/train_dreambooth.py                  --stop_text_encoder_training=350      --image_captions_filename      --train_only_unet      --save_starting_step=500      --save_n_steps=0      --Session_dir=/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6      --pretrained_model_name_or_path="/content/stable-diffusion-v1-5"      --instance_data_dir="/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6/instance_images"      --output_dir="/content/models/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6"      --captions_dir="/content/gdrive/MyDrive/Fast-Dreambooth/Sessions/elinckus-fast-dremlikephotov2-2100(14)-350-5e6-1e6/captions"      --instance_prompt=""      --seed=610410      --resolution=512      --mixed_precision=fp16      --train_batch_size=1      --gradient_accumulation_steps=1 --gradient_checkpointing      --use_8bit_adam      --learning_rate=5e-06      --lr_scheduler="polynomial"      --lr_warmup_steps=0      --max_train_steps=2100'

Something went wrong

**Comment by TheLastBen** [comment]
It is probably caused by the session name, try a simple session name and see if the error persists

**Comment by inck86** [comment]
> It is probably caused by the session name, try a simple session name and see if the error persists, and make sure you didn't accidentally change the code in the training cell.

Thank you! That helped.
You cannot use "(" in the session name.
